packages/server-wallet/e2e-testing/wallet-load-node.ts

```diff
@@ -197,31 +203,74 @@ export class WalletLoadNode {
    * These are the various handlers for different step types
    * TODO: Typing should not use any type for the request
```

Comment:
Nice! Locally I'm seeing this error when I try this but I could be using it wrong?

```
Argument of type 'Step' is not assignable to parameter of type 'never'.
  The intersection 'CreateChannelStep & CreateLedgerChannelStep & CloseChannelStep & UpdateChannelStep' was reduced to 'never' because property 'type' has conflicting types in some constituents.
    Type 'CreateLedgerChannelStep' is not assignable to type 'never'.
```

Reply:
Oh yeah I see that too now!

Reply:
Might be going down a rabbit hole here, but this seems to work along with what I wrote before:

```ts
  private async handleStep(step: Step): Promise<ObjectiveDoneResult | UpdateChannelResult> {
    switch (step.type) {
      case 'CreateChannel':
        return this.stepHandlers[step.type](step);
      case 'CloseChannel':
        return this.stepHandlers[step.type](step);
      case 'UpdateChannel':
        return this.stepHandlers[step.type](step);
      case 'CreateLedgerChannel':
        return this.stepHandlers[step.type](step);
    }
  }
```

Not sure if it's worth it, though!

Reply:
Going to punt this to a new issue!